go.mod

```diff
@@ -26,23 +25,25 @@ require (
 	github.com/teambition/rrule-go v1.8.2
 	go.uber.org/multierr v1.7.0
 	go.uber.org/zap v1.24.0
-	golang.org/x/net v0.8.0
-	golang.org/x/oauth2 v0.1.0
+	golang.org/x/net v0.9.0
+	golang.org/x/oauth2 v0.7.0
```

Comment:
This seems to align with go-github v52 👍 

https://github.com/google/go-github/blob/v52.0.0/go.mod